progress/progress.go

```diff
@@ -149,13 +158,13 @@ func (b *Bar) Done() {
 	if b.ticker != nil {
 		b.ticker.Stop()
 		b.ticker = nil
-		b.overwrite("") // write unindented
+		// b.overwrite("") // write unindented
 	}
 }
 
 // Write a simple line status update.
 func (b *Bar) write() {
-	fmt.Fprintf(b.out, "%v/%v %v\n", b.index, b.total, b.text)
+	fmt.Fprint(b.out, b.format())
```

Comment:
This looks like a good opportunity to just allow an instance of Bar to always be capable of printing it's current state to String, by using the standard method `.String()`.   If `format` is renamed as such, this line I believe cold become:
```suggestion
	fmt.Fprintln(b.out, b)
```